ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -355,13 +350,29 @@ trait AbstractHttpServiceIntegrationTestFuns
     VA.record(Ref.Identifier assertFromString "none:Iou:Iou", iouT)
   }
 
+  protected[this] object TplID {
```

Comment:
I would prefer the slight change to `TpId` (`Id` instead of `ID`, I personally like that for being easier to type).

Reply:
c62a52fcec6de5ad4df9e4191f7dc18b2e29bc91